File: Class/1117/GridView1/app/src/main/java/com/example/gridview1/MainActivity.java
package com.example.gridview1;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.widget.GridView;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        setTitle("그리드뷰 영화 포스터");

        final GridView gv=(GridView) findViewById(R.id.gridView01);
        MyGridAdapter gAdapter=new MyGridAdapter(this);
        gv.setAdapter(gAdapter);
    }
}

File: Kotlin/week1/TextViewKt/app/src/main/java/com/example/textviewkt/MainActivity.kt
package com.example.textviewkt

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import kotlinx.android.synthetic.main.activity_main.*


class MainActivity : AppCompatActivity() {


    override fun onCreate(savedInstanceState: Bundle?) {    //앱이 최초 실행되었을 때 수행
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main) //xml 화면 view를 연결한다.

        btn_getText.setOnClickListener{ //edit text에 입력되어 있는 값을 가지고 와서 텍스트 뷰에 보여준다.
            var resultText=edit_id.text.toString() //edit text에 입력되어 있는 값
            tv_result.setText(resultText) //입력된 값을 텍스트뷰에 set해줌

        }
    }
}